### DISCUSSION TITLE
How could I add fonts in Toolbar

### DISCUSSION THREAD

**Discussion by yyhan0915** [facebook/lexical#2397] [Q&A]
Dear Lexical team,

is there way to add some fonts in Toolbar?
I checked playground but no example code on adding fonts.

Please kindly give me example.

BR,

**Comment by acywatson** [comment]
We have fonts in the playground: https://github.com/facebook/lexical/blob/5caf9231d456cadeb0f83fb69602b3bc6e433710/packages/lexical-playground/src/plugins/ToolbarPlugin.tsx#L907

**Comment by rahuldesktop** [comment]
How to load custom fonts? I have tried adding custom google fonts but some of them are not working, I have also created index.css file 
const FONT_FAMILY_OPTIONS: [string, string][] = [
  ["Arial", "Arial"],
  ["Courier New", "Courier New"],
  ["Georgia", "Georgia"],
  ["Times New Roman", "Times New Roman"],
  ["Trebuchet MS", "Trebuchet MS"],
  ["Verdana", "Verdana"],
  ["Comic Sans MS", "Comic Sans MS"],
  ["Calibri", "Calibri"],
  ["Poppins", "Poppins"],
  ["Alegraya", "Alegraya"], // Not working
  ["Bree Sarif", "Bree Sarif"], // Not working
  ["Cambria", "Cambria"], // Not working
  ["Impact", "Impact"],
  ["Lexend", "Lexend"], // Not working
  ["Lobster", "Lobster"], // Not working
  ["Lora", "Lora"], // Not working
  ["Premanent Marker", "PERMANENT MARKER"], // Not working
  ["Ultra", "Ultra"], // Not working
  ["Reenie Beanie", "Reenie Beanie"]
];
